mne_bids/tests/test_read.py

```diff
@@ -266,7 +266,7 @@ def test_line_freq_estimation():
     somato_raw.set_channel_types({somato_raw.ch_names[i]: 'bio'
                                   for i in range(len(somato_raw.ch_names))})
     somato_raw = _handle_info_reading(sidecar_fname, somato_raw, verbose=True)
-    assert somato_raw.info['line_freq'] is None
+    assert somato_raw.info['line_freq'] == -1
```

Comment:
I would object here. I would keep it None

Reply:
The bids-validator throws an error if it's not an integer, you can't write None or -1 to the file, that's why I did it that way